resonantLimits/get_limits_res.sh

```diff
@@ -168,50 +169,50 @@ IDENTIFIER=".test_${SIGNAL}_${VAR}"
 if [ ${MODE} == "separate" ]; then
 	
     for i in "${!CHANNELS[@]}"; do
-	card_dir="${LIMIT_DIR}/cards_${TAG}_${CHANNELS[$i]}"
-	cd ${card_dir}
-
-	printf "Processing categories in parallel for channel ${CHANNELS[$i]} "
-	printf "(mode ${MODE}, var ${VAR})...\n"
+		card_dir="${LIMIT_DIR}/cards_${TAG}_${CHANNELS[$i]}"
+		cd ${card_dir}
 
-	out_="combined_out"
-	for sel in ${SELECTIONS[@]}; do
-	    cat_dir="${card_dir}/${sel}_${VAR}"
-	    out_dir="${cat_dir}/${out_}"
-	    mkdir -p "${out_dir}"
-	done
+		printf "Processing categories in parallel for channel ${CHANNELS[$i]} "
+		printf "(mode ${MODE}, var ${VAR})...\n"
+		
+		out_="combined_out"
+		for sel in ${SELECTIONS[@]}; do
+			cat_dir="${card_dir}/${sel}_${VAR}"
+			out_dir="${cat_dir}/${out_}"
+			mkdir -p "${out_dir}"
+		done
 
-	cat_dir_parallel="${card_dir}/{2}_${VAR}"
-	out_dir_parallel="${cat_dir_parallel}/${out_}"
+		cat_dir_parallel="${card_dir}/{2}_${VAR}"
+		out_dir_parallel="${cat_dir_parallel}/${out_}"
 	
-	# parallellize across the mass and category
-	proc="${SIGNAL}_${VAR}_{1}"
-	in_txt="${cat_dir_parallel}/comb.${proc}.txt"
-	out_log="${out_dir_parallel}/comb.${proc}.log"
-
-	parallel rm -f -- ${out_log} ::: ${MASSES[@]} ::: ${SELECTIONS[@]}
-
-	if [ ${#CATEGORIES_BOOST[@]} -ne 0 ]; then
-		parallel -j $((`nproc` - 1)) \
-				 combine -M AsymptoticLimits ${in_txt} \
-				 -n ${IDENTIFIER}_{2} \
-				 --run blind \
-				 --noFitAsimov \
-				 --freezeParameters SignalScale \
-				 -m {1} ">" ${out_log} ::: ${MHIGH[@]} ::: ${CATEGORIES_BOOST[@]}
-	fi
-
-	if [ ${#CATEGORIES_NOBOOST[@]} -ne 0 ]; then
-		parallel -j $((`nproc` - 1)) \
-				 combine -M AsymptoticLimits ${in_txt} \
-				 -n ${IDENTIFIER}_{2} \
-				 --run blind \
-				 --noFitAsimov \
-				 --freezeParameters SignalScale \
-				 -m {1} ">" ${out_log} ::: ${MASSES[@]} ::: ${CATEGORIES_NOBOOST[@]}
-	fi
+		# parallellize across the mass and category
+		proc="${SIGNAL}_${VAR}_{1}"
+		in_txt="${cat_dir_parallel}/comb.${proc}.txt"
+		out_log="${out_dir_parallel}/comb.${proc}.log"
+
+		parallel rm -f -- ${out_log} ::: ${MASSES[@]} ::: ${SELECTIONS[@]}
+
+		if [ ${#CATEGORIES_BOOST[@]} -ne 0 ]; then
+			parallel -j $((`nproc` - 1)) \
+					 combine -M AsymptoticLimits ${in_txt} \
+					 -n ${IDENTIFIER}_{2} \
+					 --run blind \
+					 --noFitAsimov \
+					 --freezeParameters SignalScale \
+					 -m {1} ">" ${out_log} ::: ${MHIGH[@]} ::: ${CATEGORIES_BOOST[@]}
+		fi
+		
+		if [ ${#CATEGORIES_NOBOOST[@]} -ne 0 ]; then
+			parallel -j $((`nproc` - 1)) \
+					 combine -M AsymptoticLimits ${in_txt} \
+					 -n ${IDENTIFIER}_{2} \
+					 --run blind \
+					 --noFitAsimov \
+					 --freezeParameters SignalScale \
+					 -m {1} ">" ${out_log} ::: ${MASSES[@]} ::: ${CATEGORIES_NOBOOST[@]}
+		fi
 
-	cd -
+		cd -
```

Comment:
is there anything other that the indentation that changed here? if not can you please revert it?

Reply:
I've uniformized the indentation, that was indeed off in some blocks. However this script got a big update, since I added ```elif``` conditions to support the new limit year combination.